internal/store/utils.go

```diff
@@ -172,6 +176,26 @@ func isPrefixedNativeResource(name v1.ResourceName) bool {
 	return strings.Contains(string(name), v1.ResourceDefaultNamespacePrefix)
 }
 
+// createAnnotationKeysValues takes in passed kubernetes annotations and allowed list in kubernetes label format
+// it returns only those allowed annotations that exist in the list converting them to Prometheus labels.
+func createAnnotationKeysValues(allKubeAnnotations map[string]string, allowList []string) ([]string, []string) {
```

Comment:
What is the differ4ence between this function and `createLabelKeysValues`? Seems like if we passed a string key of `annotation ` and called `mapToPrometheusLabels ` with that key it would work correctly. Or at least refactor the common things into one function, wdyt?

Reply:
Agreed